pysr/sr.py

```diff
@@ -949,7 +949,7 @@ def __init__(
     @classmethod
     def from_file(
         cls,
-        equation_file: PathLike,
+        equation_file: Union[str, Path],
```

Comment:
```suggestion
        equation_file: PathLike,
```
`PathLike` is already equal to `Union[str, Path]` :) (It's just a convenient/easier-to-read name)